web-ng/src/app/components/map/ground-pin-svg-template.ts

```diff
@@ -0,0 +1,28 @@
+export const svgTemplate = `<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Design nit: Could we make this a function that returns the SVG with the appropriate color already set? This way the template placeholder doesn't need to be exposed outside this file.